README.md

```diff
@@ -41,6 +42,14 @@ secrets from Conjur.
 
 You can find our official distributable releases on Puppet Forge under [`cyberark/conjur`](https://forge.puppet.com/cyberark/conjur).
 
+## Certification level
```

Comment:
With the addition of the certification level section, the caveat about Puppet v6 support being community level under "Setup Requirements" should be removed.

Also, you may want to move the section about "Certified Version" below up to this area, so that there is only a single part of the README dedicated to certification level.  @izgeri, WDYT?